pkg/services/ngalert/state/historian/annotation.go

```diff
@@ -56,19 +55,19 @@ func (h *AnnotationStateHistorian) RecordState(ctx context.Context, rule *ngmode
 			return
 		}
 
-		query := &models.GetDashboardQuery{
-			Uid:   dashUid,
-			OrgId: rule.OrgID,
-		}
-
-		err = h.dashboards.GetDashboard(ctx, query)
+		dashID, err := h.dashboards.getId(ctx, rule.OrgID, dashUid)
```

Comment:
Looks more consistent 😄 

```suggestion
		dashID, err := h.dashboards.getID(ctx, rule.OrgID, dashUid)
```